### PR TITLE
Adding new Volume

### PR DESCRIPTION
**Pull request by pasc-ed**
New GP3 volume added and attached to the web-server

### CODE REVIEW

File: infrastructure/ec2.tf [hunk 1]
```diff
@@ -9,7 +7,7 @@ resource "aws_instance" "web_server" {
   subnet_id              = data.aws_subnet.public.id
 
   tags = {
-    Name = "web-server-${count.index + 1}"
+    Name = "web-server-1"
```
Comment (denissstar) [inline infrastructure/ec2.tf]: Do we need a number here

**Review by denissstar — CHANGES_REQUESTED** [review]
Also please delete emty file `new_file`

File: infrastructure/ec2.tf (hunk 1 above)
Comment (pasc-ed) [reply infrastructure/ec2.tf]: Good point, I'll remove it.

File: infrastructure/ec2.tf [hunk 2]
```diff
@@ -42,6 +40,22 @@ resource "aws_security_group" "web_server" {
   }
 
   tags = {
-    Name = "ansible-web-server"
+    Name = "ansible-web-server-1"
```
Comment (denissstar) [inline infrastructure/ec2.tf]: same unneeded index here

**Review by denissstar — APPROVED** [review]
LGTM  💯

**Pull request merged**